package.json

```diff
@@ -33,11 +33,16 @@
     "diacritics": "^1.3.0",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "lrc-kit": "1.1.1",
     "spotify-web-api-node": "^4.0.0",
     "typescript": "^3.9.4"
   },
   "devDependencies": {
     "ts-node-dev": "^1.0.0-pre.43",
     "tslint": "^5.20.0"
+  },
+  "engines" :{
+    "node": ">=18.12.1"
```

Comment:
Not sure if it will break your stuff, but you can drop this down to `>=16.13.1` (I'm running locally on some old stuff and it works 😄)